Index: src/com/kh/mini/dao/ProductDAO.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/com/kh/mini/dao/ProductDAO.java b/src/com/kh/mini/dao/ProductDAO.java
new file mode 100644
--- /dev/null	(date 1677746182577)
+++ b/src/com/kh/mini/dao/ProductDAO.java	(date 1677746182577)
@@ -0,0 +1,50 @@
+package com.kh.mini.dao;
+
+import com.kh.mini.util.Common;
+import com.kh.mini.vo.OrderList;
+import com.kh.mini.vo.Products;
+
+import java.sql.Connection;
+import java.sql.Date;
+import java.sql.ResultSet;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+
+/*
+장바구니 표시 데이터 : 상품이름 , 고객아이디, 개수
+ */
+public class ProductDAO {
+    public List<Products> listEmp() {
+        Connection conn = null;
+        Statement stmt = null;
+        ResultSet rs = null;
+        List<Products> list = new ArrayList<>();
+        try {
+            conn = Common.getConnection();
+            stmt = conn.createStatement();
+            String query = "SELECT * FROM PRODUCTS";
+            rs = stmt.executeQuery(query);
+
+            while(rs.next()) {
+                int id = rs.getInt("PRODUCT_ID");
+                String name = rs.getString("PRODUCT_NAME");
+                String color = rs.getString("COLOR");
+                int price = rs.getInt("PRICE");
+                String made = rs.getString("MADE_IN");
+
+
+                    Products vo = new Products(id, name, color, price, made);
+
+                list.add(vo);
+            }
+            Common.close(rs);
+            Common.close(stmt);
+            Common.close(conn);
+
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+        return list;
+    }
+}
